src/lib.rs

```diff
@@ -43,6 +43,16 @@ pub struct ToipeError {
     pub msg: String,
 }
 
+impl ToipeError {
+    /// Prefixes a ToipeError's msg with 'msg_context'
```

Comment:
Nitpick: mentioning `ToipeError` here is redundant. This can be:
```suggestion
    /// Prefixes the message with a context
```